ajax/libs/later/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "later",
+  "version": "1.1.6",
+  "description": "Determine later (or previous) occurrences of recurring schedules",
+  "keywords": [
+    "schedule",
+    "occurrences",
+    "recur",
+    "cron"
+  ],
+  "author": "BunKat <bill@bunkat.com>",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bunkat/later.git"
+  },
+  "main": "index.js",
+  "browserify": "index-browserify.js",
+  "jam": {
+    "main": "later.js",
+    "shim": {
+      "exports": "later"
+    }
+  },
+  "devDependencies": {
+    "smash": "~0.0.8",
+    "mocha": "*",
+    "should": ">=0.6.3",
+    "jslint": "*",
+    "uglify-js": "*",
+    "benchmark": "*"
+  },
+  "license": "MIT",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha test/**/*-test.js"
+  },
+  "npmName": "later",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
+      "files": [
+        "**/*"
```

Comment:
So I just wanted to add files that are really necessary. Do I have to add these files OR change the package.json?